modules/ROOT/pages/introduction/introduction-to-anypoint-connectors.adoc

```diff
@@ -0,0 +1,122 @@
+
+* HTTP Listener is a connector that listens to HTTP requests. You can configure an HTTP Listener to start a flow when it receives specified requests.
+* Scheduler is a core component that starts a flow when a time-based condition is met. You can configure a Scheduler to start a flow at regular intervals, or you can specify a more flexible cron expression to start the flow.
+
+For more information about the HTTP Listener and the Scheduler component, see the xref:http/http-connector.adoc[HTTP Connector User Guide] and xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)].
+
+[[operations]]
+=== Operations
+
+Most connectors have operations that execute API calls or other actions on their associated system, database, or protocol. For example, you can use Anypoint Connector for Workday (Workday Connector) to create a position request in Workday or add a fund to the financial management service. Likewise, you can use Anypoint Connector for VMs (VM Connector) to consume messages from and publish messages to an asynchronous queue.
+
+To see a list of operations for a connector, see the Reference Guide for that connector.
+
+== Access Connectors and Related Assets
+
+Anypoint Exchange provides access to all publicly-available connector assets including connectors, templates, and examples.
+
+=== Connectors in Exchange
+
+You can use Exchange as a starting point for discovering all or a subset of MuleSoft-built connectors.
+
+For information about locating connectors in Exchange, see <<introduction/discover-add-connectors.adoc#Discover and Add Connectors>>.
+
+=== Templates in Exchange
+
+Templates provide starting points for creating Studio projects. Many templates include one or more connector configurations.  For example, the S/4HANA Cloud to Salesforce Partner Migration template helps you migrate partners from an SAP S/4HANA Cloud to a Salesforce instance.
+
+Once you configure a template in Flow Designer or Studio, you can customize it to meet your needs.
+
+For information about locating and using templates, see <<introduction/discover-add-templates.adoc#Discover and Add Templates to Flow Designer or Studio>>.
+
+=== Examples in Exchange
+
+Examples are apps that demonstrate how to handle a use case or solution. Many examples use one or more connectors to accomplish required tasks.
+
+For example, the Authenticating Salesforce using OAuth 2.0 example shows you how to connect to Salesforce using OAuth as the security protocol. It uses the Salesforce Connector to enable OAuth authentication before performing the integration process.
+
+For information about locating and using examples, see <<introduction/discover-add-examples.adoc#Discover and Add Examples to Studio>>.
+
+== Connector Support Categories
+
+Anypoint Connectors have the following categories of support:
+
+* Select connectors are available to anyone. However, you must have an Anypoint Platform subscription to access MuleSoft support for these connectors.
```

Comment:
```suggestion
* Select connectors are available to anyone. 
However, you must have an Anypoint Platform subscription to access MuleSoft support for these connectors.
```

Reply:
Keeping this one together since it is so tightly linked with the first sentence.